mobile/test/common/integration/rtds_integration_test.cc

```diff
@@ -85,17 +68,16 @@ TEST_P(RtdsIntegrationTest, RtdsReload) {
   auto some_rtds_layer = TestUtility::parseYaml<envoy::service::runtime::v3::Runtime>(R"EOF(
     name: some_rtds_layer
     layer:
-      envoy.reloadable_features.test_feature_false: False
-      envoy.reloadable_features.test_feature_true: False
+      envoy.reloadable_features.skip_dns_lookup_for_proxied_requests: True
   )EOF");
   sendDiscoveryResponse<envoy::service::runtime::v3::Runtime>(
       Config::TypeUrl::get().Runtime, {some_rtds_layer}, {some_rtds_layer}, {}, "1");
   // Wait until the RTDS updates from the DiscoveryResponse have been applied.
   ASSERT_TRUE(waitForCounterGe(load_success_counter, load_success_value + 1));
 
   // Verify that the Runtime config values are from the RTDS response.
-  EXPECT_FALSE(Runtime::runtimeFeatureEnabled("envoy.reloadable_features.test_feature_false"));
-  EXPECT_FALSE(Runtime::runtimeFeatureEnabled("envoy.reloadable_features.test_feature_true"));
+  EXPECT_TRUE(Runtime::runtimeFeatureEnabled(
+      "envoy.reloadable_features.skip_dns_lookup_for_proxied_requests"));
 }
 
```

Comment:
Can we also add a test to ensure we throw an error when addRtdsLayer is called without calling setAggregatedDiscoveryService?